### DISCUSSION TITLE
Integration with Tesseract Motion Planning Framework

### DISCUSSION THREAD

**Discussion by acbuynak** [ros-industrial/reach_ros2#21] [Ideas]
More of a discussion question..

I have a system that will exclusively use Tesseract for motion planning and would like to integrate `reach_ros2`. Has anyone ever migrated the `reach_ros2` evaluators to use Tesseract plugins instead of MoveIt? 

Or would it be better advised to just run Tesseract and MoveIt in parallel?

**Comment by marip8** [comment]
I've had Tesseract integration on my to-do list for a while, but I've never gotten around to it. I think it would be a good idea, but I would still like to support both MoveIt and Tesseract. Depending on what you want to implement, it might make sense to make the Tesseract integration a separate repository
